src/main/java/org/dependencytrack/tasks/InternalComponentIdentificationTask.java

```diff
@@ -72,6 +97,12 @@ private void analyze() throws Exception {
 
             List<Component> components = fetchNextComponentsPage(pm, null);
             while (!components.isEmpty()) {
+                //Extend the lock by 5 min everytime we have a page.
+                //We will get max 500 components in a page
+                //Reason of not extending at the end of loop is if it does not have to do much,
+                //It might finish execution before lock could be extended resulting in error
+                LOGGER.debug("extending lock of internal component identification by 5 min");
+                LockExtender.extendActiveLock(Duration.ofMillis(300000), ZERO);
```

Comment:
The comment above mentions minutes as time unit, I think this would make it more readable:

```suggestion
                LockExtender.extendActiveLock(Duration.ofMinutes(5), ZERO);
```